etc/config.py

```diff
@@ -109,3 +121,10 @@
         'url': 'https://yale-som-execed.onlinecampus.getsmarter.com/admin/plugins.php',
     }
 ]
+
+try:
+    # pylint: disable=unused-wildcard-import,wildcard-import
```

Comment:
Should we leave this commented out code in? Not sure if this is intentional

Reply:
Intentional - this is a flag to disable the warning in the linter. Wildcard required in this instance.